addons/account/wizard/account_invoice_send.py

```diff
@@ -41,6 +41,14 @@ def default_get(self, fields):
         })
         return res
 
+    @api.model
+    def create(self, vals):
+        res = super().create(vals)
+        a = len(res)
+        if (a==1):
+            res.composer_id.template_id = res.template_id.id
+        return res
```

Comment:
```suggestion
@api.model
def create(self, vals):
    ctx = {}

    # Make sure the composer is created with the right template_id since it's an inherits.
    if vals.get('template_id'):
        ctx['default_template_id'] = vals['template_id']

    return super(AccountInvoiceSend, self.with_context(**ctx)).create(vals)
```